docs/flows.md

```diff
@@ -112,19 +117,17 @@ observability).
 
 ### Cloud Function for Firebase
 
-To use flows with Cloud Functions for Firebase use the `firebase` plugin and replace `defineFlow` with `onFlow`.
+To use flows with Cloud Functions for Firebase use the `firebase` plugin, replace `defineFlow` with `onFlow` and include an `authPolicy`.
 
 ```js
 import { onFlow, noAuth } from '@genkit-ai/firebase/functions';
 
-export const jokeFlow = onFlow(
+export const menuSuggestionFlow = onFlow(
   {
-    name: 'jokeFlow',
-    inputSchema: z.string(),
-    outputSchema: z.string(),
+    name: 'menuSuggestionFlow',
     authPolicy: noAuth(),
```

Comment:
Add a comment that this should probably be replaced by a "real" auth policy?

Reply:
Decided to just replace this with a real auth policy.